apps/bfd-model/bfd-model-rif/src/main/resources/db/scripts/Query_synthetic_id_ranges.sql

```diff
@@ -0,0 +1,102 @@
+-- Find the beneficiaries.bene_id range to avoid collisions with synthetic data.
```

Comment:
I suggest having a paragraph at the top of the file here to explain the purpose of this script and give an overview of what follows. Something along these lines:

This script contains queries that should be run in prod-sandbox prior to generating new synthea data in order to ensure that any identifiers that are generated do not conflict with existing synthetic data. In the case of claimId (for all non-partd event tables) and pde_id (for the partd event table) new synthetic data is appended below the minimum value currently present. In the case of beneficiary IDs (bene_id) and claim group IDs (clm_grp_id) it is necessary to use a more complex query to extract all currently occupied ranges of IDs and find a gap that is big enough for the new generated data.